Quaver.API/Maps/Qua.cs

```diff
@@ -1056,12 +1008,10 @@ public static void RestoreDefaultValues(Qua qua)
         /// <exception cref="ArgumentException"></exception>
         private static void AfterLoad(Qua qua, bool checkValidity)
         {
-            if (checkValidity)
-            {
-                var errors = qua.Validate();
-                if (errors.Count > 0)
-                    throw new ArgumentException(string.Join("\n", errors));
-            }
+            var errors = qua.Validate();
+
```

Comment:
I prefer this section the old way so that the computation of checking validity isn't performed if not  needed.

Reply:
```csharp
if (checkValidity && qua.Validate() is var errors && errors.Count > 0)
```

Reply:
I'll just merge with upstream.

Reply:
Fixed.